docs/docs/connect/data-source/postgres.md

```diff
@@ -48,8 +54,15 @@ For more details, see our [Dev/Prod setup docs](/connect/templating).
 
 ## Cloud deployment
 
-Once a project with a PostgreSQL source has been deployed, Rill requires you to explicitly provide the connection string using the following command:
+Once a project with a MySQL source has been deployed, Rill requires you to explicitly provide the connection string using the following command:
 
 ```
 rill env configure
 ```
+
+
+:::tip Did you know?
+
+If you've already configured credentials locally (in your `<RILL_PROJECT_DIRECTORY>/.env` file), you can use `rill env push` to [push these credentials](/connect/credentials.md#rill-env-push) to your Rill Cloud project. This will allow other users to retrieve and reuse the same credentials automatically by running `rill env pull`.
+
+:::
```

Comment:
I'm not so sure, but it feels a little wrong to split this from the `rill env pull` documentation that's just a bit higher up